### DISCUSSION TITLE
Possibility of using PostgreSQL window functions without using raw sql

### DISCUSSION THREAD

**Discussion by edilson** [prisma/prisma#6368] [Q&A]
Is it possible to use window functions without having to do it with raw sql queries? I searched in the docs but couldn't find anything related.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
